### PR TITLE
docs: first draft on how to write a connector

### PR DESCRIPTION
**Pull request by Lacritz**

### CODE REVIEW

File: docs/WRITING-A-CONNECTOR.mdx [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+### Building Blocks
+
+The connector project usually is build by the following 5 blocks:
+
+- <dfn>Processors</dfn>: Explained previously.
+- <dfn>Document</dfn>: A Document is a searchable Record which has to have a globally unique identifier. Even amongst
+  documents of different types.
+- <dfn>Record</dfn>: A Record is a specifically annotated class which then can be stored in the backend.
+- <dfn>KeyNode</dfn>: A KeyNode is basically an identifier in the terminology of a connector. KeyNodes are not stored in
```
[the remark this answers is not in the document]
Comment (zeljkobekcic) [reply docs/WRITING-A-CONNECTOR.mdx]: I rewrote this section in terms of the `ContainerNode`, but still mentioning these two terms as how they are usually called.

File: docs/WRITING-A-CONNECTOR.mdx [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+  the backend.
+- <dfn>KeyNodeContainer</dfn>: A KeyNodeContainer is a collection of KeyNodes. Usually you retrieve a collection of Ids
```
[the remark this answers is not in the document]
Comment (zeljkobekcic) [reply docs/WRITING-A-CONNECTOR.mdx]: s.a.

File: templates/docs/WRITING-A-CONNECTOR.mdx [hunk 3]
```diff
@@ -31,26 +33,26 @@ Next we want to set up the three projects you previously pulled.
...
+The API should instantly become ready on http://127.0.0.1:8000. You can easily verify it by visiting
+http://127.0.0.1:8000/docs. If everything went well, the FastAPI Swagger UI should become available.
 
 ![](img/fast_api_swagger_ui.png)
 
-### Sample API Client
+### Sample API Client and Sample API Connector
 
-At best import it with your IDE of choice. More on that later.
+These two projects use gradle with the `idea` plugin. So you can easily import them with IntelliJ.
```
Comment (Lacritz) [inline templates/docs/WRITING-A-CONNECTOR.mdx:50]: ... by using `./gradlew openIdea`

Comment (zeljkobekcic) [reply templates/docs/WRITING-A-CONNECTOR.mdx]: Done!

**Pull request merged**